src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/InferParameterBindingInfoConvention.cs

```diff
@@ -29,41 +32,27 @@ public class InferParameterBindingInfoConvention : IControllerModelConvention
             _modelMetadataProvider = modelMetadataProvider ?? throw new ArgumentNullException(nameof(modelMetadataProvider));
         }
 
-        /// <summary>
-        /// Gets or sets a value that determines if model binding sources are inferred for action parameters on controllers is suppressed.
-        /// </summary>
-        public bool SuppressInferBindingSourcesForParameters { get; set; }
+        internal bool AllowInferringBindingSourceForCollectionTypesAsFromQuery { get; set; }
```

Comment:
Why is this not `public`?

Reply:
This convention type is new to 2.2, so users wouldn't need a compat flag.